what4/src/What4/Interface.hs

```diff
@@ -601,7 +601,7 @@ class ( IsExpr (SymExpr sym), HashableF (SymExpr sym)
   --   @div@ and @mod@ are the unique Eucledian division operations satisfying the
```

Comment:
There's another typo here that I'm embarrassed that I didn't point out before: `Eucledian` should be `Euclidean`.